app-header/test/app-header.html

```diff
@@ -48,7 +48,7 @@
   <test-fixture id="testHeader">
     <template>
       <div>
-        <app-header>
+        <app-header class="header">
```

Comment:
Hmm... this sounds like a bug in Polymer.  Users should be able to override the :host rule.  Let me try to reproduce it in a simple testcase.